pkg/cloud/libvirt/actuators/machine/utils/domain.go

```diff
@@ -497,9 +497,6 @@ func CreateDomain(name, ignKey, volumeName, hostName, networkInterfaceName, netw
 
 	glog.Infof("setDisks")
 	VolumeKey := baseVolumePath + volumeName
```

Comment:
could you also update this to use https://golang.org/pkg/path/#Join

Reply:
Done